Changes

```diff
@@ -68,6 +68,11 @@ _______________
 
 ### Bug fixes:
 
+- #11040: Silences false data race observed between caml_shared_try_alloc
+  and oldify. Introduces macros to call tsan annotations which help annotate
+  a ``happens before'' relationship.
+  (Olivier Nicole, review by Gabriel Scherer, Hari Hara Naveen S)
```

Comment:
The usual format is
```
(Hari Hara Naveen S and Olivier Nicole, review by Gabriel Scherer)
```
Could you also add @dustanddreams as a reviewer?

Reply:
So in full:

```
(Hari Hara Naveen S and Olivier Nicole,
 review by Gabriel Scherer and Miod Vallat)
```